File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADMS2015U00SystemHospitalInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
	public class ADMS2015U00SystemHospitalInfo
	{
		private String _admsGroupSystemId;
		private String _systemSeq;
		private String _hospCode;
		private String _selectFlg;
		private String _systemId;
		private String _sysNm;
		private String _dataRowState;

		public String AdmsGroupSystemId
		{
			get { return this._admsGroupSystemId; }
			set { this._admsGroupSystemId = value; }
		}

		public String SystemSeq
		{
			get { return this._systemSeq; }
			set { this._systemSeq = value; }
		}

		public String HospCode
		{
			get { return this._hospCode; }
			set { this._hospCode = value; }
		}

		public String SelectFlg
		{
			get { return this._selectFlg; }
			set { this._selectFlg = value; }
		}

		public String SystemId
		{
			get { return this._systemId; }
			set { this._systemId = value; }
		}

		public String SysNm
		{
			get { return this._sysNm; }
			set { this._sysNm = value; }
		}

		public String DataRowState
		{
			get { return this._dataRowState; }
			set { this._dataRowState = value; }
		}

		public ADMS2015U00SystemHospitalInfo() { }

		public ADMS2015U00SystemHospitalInfo(String admsGroupSystemId, String systemSeq, String hospCode, String selectFlg, String systemId, String sysNm, String dataRowState)
		{
			this._admsGroupSystemId = admsGroupSystemId;
			this._systemSeq = systemSeq;
			this._hospCode = hospCode;
			this._selectFlg = selectFlg;
			this._systemId = systemId;
			this._sysNm = sysNm;
			this._dataRowState = dataRowState;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADMS2015U00SystemHospitalInfo.java
package nta.med.data.model.ihis.adma;

import java.math.BigDecimal;

public class ADMS2015U00SystemHospitalInfo {
	private Integer admsGroupSystemId;
	private Integer sysSeq;
	private String hospCode;
	private BigDecimal selectFlg;
	private String systemId;
	private String sysNm;
	private String rowState;
	public ADMS2015U00SystemHospitalInfo(Integer admsGroupSystemId,
			Integer sysSeq, String hospCode, BigDecimal selectFlg,
			String systemId, String sysNm, String rowState) {
		super();
		this.admsGroupSystemId = admsGroupSystemId;
		this.sysSeq = sysSeq;
		this.hospCode = hospCode;
		this.selectFlg = selectFlg;
		this.systemId = systemId;
		this.sysNm = sysNm;
		this.rowState = rowState;
	}
	public Integer getAdmsGroupSystemId() {
		return admsGroupSystemId;
	}
	public void setAdmsGroupSystemId(Integer admsGroupSystemId) {
		this.admsGroupSystemId = admsGroupSystemId;
	}
	public Integer getSysSeq() {
		return sysSeq;
	}
	public void setSysSeq(Integer sysSeq) {
		this.sysSeq = sysSeq;
	}
	public String getHospCode() {
		return hospCode;
	}
	public void setHospCode(String hospCode) {
		this.hospCode = hospCode;
	}
	public BigDecimal getSelectFlg() {
		return selectFlg;
	}
	public void setSelectFlg(BigDecimal selectFlg) {
		this.selectFlg = selectFlg;
	}
	public String getSystemId() {
		return systemId;
	}
	public void setSystemId(String systemId) {
		this.systemId = systemId;
	}
	public String getSysNm() {
		return sysNm;
	}
	public void setSysNm(String sysNm) {
		this.sysNm = sysNm;
	}
	public String getRowState() {
		return rowState;
	}
	public void setRowState(String rowState) {
		this.rowState = rowState;
	}
	
}
